server/middleware/setUpWebSession.ts

```diff
@@ -40,5 +40,13 @@ export default function setUpWebSession(): Router {
     next()
   })
 
+  // Ensure there is always a prisoner context holder
+  router.use((req, res, next) => {
+    if (!req.session.prisonerContexts) {
```

Comment:
this is to avoid having to perform this check every time we access the context. It could also be moved to the helper function instead.